packages/vuetify/src/components/VSelect/VSelect.ts

```diff
@@ -248,6 +258,8 @@ export default baseMixins.extend<options>().extend({
       })
     },
     isMenuActive (val) {
+      this.$nextTick(() => this.onMenuActiveChange(val))
```

Comment:
why before `!val` check?

Reply:
Because the **val** check only pertains to the setting of the **isBooted** variable. The **onMenuActiveChange** method must occur every time.